src/core/worker.js

```diff
@@ -516,6 +516,10 @@ class WorkerMessageHandler {
       });
     });
 
+    handler.on("GetFieldObjects", function (data) {
+      return pdfManager.pdfDocument.getFieldObjects();
```

Comment:
I missed this previously, but please note that it's *generally not correct* to access data like this *anywhere* in this file. There may of course be exceptions, but unless you're using `pdfManager.onLoadedStream()` (which should be used very carefully/sparingly) you should generally not do this!

Please note that before data has been loaded, there's a very real risk that `MissingDataException`s are thrown during parsing and as-is this code would thus incorrectly return a rejected promise to the main-thread and break parsing; hence please change this to (and please keep it in mind for the future):
```suggestion
      return pdfManager.ensureDoc("getFieldObjects");
```

Reply:
Thx for the tip